docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,656 @@
+# Cross Chain EVM Swap
+In this section, we will demonstrate how you can use PKPs and Lit Actions to perform a cross-chain swap. There are three parts to this flow:
+
+- Understanding Lit PKPs as escrow
+- Lit Action
+- Implementing Mint/Grant/Burn with SDK
+
+## 1) Understanding Lit PKPs as escrow
+
+A Private key is meant for signing transactions which are executed on the blockchain, a network verifies if the transaction is a legitimate one and then executes it. With Lit Network we can generate a special type of private keys which can be programmed to sign transactions only when certain conditions are met. With this, we can create several escrow mechanisms, and we will look into how we can execute a cross-chain swap.
+
+### A swap example would be:
+Suppose Alice holds 100 token A on Base and Bob holds 100 token B on Chronicles and they both agree to a swap of,
+- 4 token A from Alice to Bob on Base
+- 8 token B from Bob to Alice on Chronicles
+
+We have an [example code](https://github.com/anshss/lit-evm-swap) for the below solution.
```

Comment:
prob link the full example at the very bottom of this doc?

Reply:
I've added a link to the specific file at the end as well. But for a developer who is trying to implement this, I think it would be easier to see the codebase at the start. 

Lemme know if this doesn't look good and I'll remove this one from here.

Reply:
sure, that works!